spring-components/rdf4j-spring-demo/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ ******************************************************************************
+  ~ Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Distribution License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  ~ ******************************************************************************
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.rdf4j</groupId>
+	<artifactId>rdf4j-spring-demo</artifactId>
+	<name>RDF4J: RDF4J-Spring Demo</name>
+	<version>4.0.0-SNAPSHOT</version>
+	<description>Demo of a spring-boot project using an RDF4J repo as its backend</description>
+	<properties>
+		<spring.boot.version>2.4.3</spring.boot.version>
+	</properties>
+	<!-- explicitly NOT submodule of maven module in parent folder so as to show what a self-contained, minimal pom looks like -->
+	<dependencies>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-spring</artifactId>
+			<version>4.0.0-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+	</dependencies>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-dependencies</artifactId>
+				<version>${spring.boot.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<fork>true</fork>
+					<release>11</release>
+					<encoding>utf8</encoding>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<version>${spring.boot.version}</version>
+			</plugin>
+		</plugins>
+	</build>
```

Comment:
Do any of these spring-boot plugins or dependencies need to have a CQ logged? As in a works-with CQ. Or maybe it's enough to check clearly defined: https://www.eclipse.org/projects/handbook/#ip-clearlydefined

It's a bit unclear if we need to do anything or not. There is a question about CQ for content on a project website, which is exempt because it's "separate from project content". Dunno if this would be.

> Are project websites subject to the IP Due Diligence Process?
>
> Website content is separate from project content. Project teams are expected to respect licenses for all web site content, but are not required to submit website content (including third-party libraries) for review by the IP Team.

Reply:
We use spring-boot 2.4.x in other parts of the project, so in that sense this should be fine. I am a little confused why this uses a different patch version though (the actual spring-components module uses 2.4.12).

Reply:
I'll fix that on Monday

Reply:
... I finally decided to change the demo to be a full child module of `rdf4j-spring-components`. It's just less of a hassle.